proto/prysm/v1alpha1/attestation.go

```diff
@@ -99,6 +102,20 @@ func (a *Attestation) Version() int {
 	return version.Phase0
 }
 
+// IsNil --
+func (a *Attestation) IsNil() bool {
+	return a == nil ||
+		a.Data == nil ||
+		a.Data.Source == nil ||
+		a.Data.Target == nil ||
+		a.AggregationBits == nil
+}
+
+// IsAggregated --
+func (a *Attestation) IsAggregated() bool {
+	return a.AggregationBits.Count() > 1
+}
```

Comment:
I created these two methods because I ran into import cycles when trying to use existing helpers. That aside, I think having such methods is nicer.